<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="slider_horizontal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_vertical">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>205</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_vertical">
      <property name="text">
       <string>Vertical Value Here</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_horizontal">
      <property name="text">
       <string>Horizontal Value Here</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_vInfo">
          <property name="text">
           <string>V_Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_vValue">
          <property name="text">
           <string>V_Range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_hInfo">
          <property name="text">
           <string>H_Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_hValue">
          <property name="text">
           <string>H_Range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDial" name="dial_Test">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>181</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="lbl_dial">
      <property name="text">
       <string>Dial Value Here</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_dInfo">
        <property name="text">
         <string>D_Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dValue">
        <property name="text">
         <string>D_Range</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
